src/app-layer-htp-file.c

```diff
@@ -85,9 +85,9 @@ int HTPFileOpen(HtpState *s, HtpTxUserData *tx, const uint8_t *filename, uint16_
  * @param[in] rawvalue
  * @param[out] range
  *
- * @return HTP_OK on success, HTP_ERROR on failure.
+ * @return HTP_STATUS_OK on success, HTP_STATUS_ERROR on failure.
  */
-int HTPParseContentRange(bstr *rawvalue, HTTPContentRange *range)
+int HTPParseContentRange(const bstr *rawvalue, HTTPContentRange *range)
```

Comment:
Can the adding of `const` be in another previous commit ?

Reply:
Like util-print and others as well ;-)

Reply:
Maybe. I can try to separate it out. Dropping the `const` sometimes generates a lot of warnings, which will make `-Werror` unhappy in some of the pipelines.

Reply:
Please do not drop the `const` :-)

I just think that this can be a first commit, independent libhtp_rs

Reply:
Sure, I can have a look and see what is possible.